Assets/Scripts/Ship/PlayerController.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PlayerController : MonoBehaviour
+{
+    [SerializeField]
+    private float trust = 34000;
+
+    [SerializeField]
+    private float turningSpeed = 1;
+
+    Rigidbody rb;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        // Get the components we need
+        rb = GetComponent<Rigidbody>();
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        // Rotate the ship using the Horizontal input axis
+        RotatePlayer(turningSpeed, Input.GetAxis("Horizontal"));
```

Comment:
Heel goed, je gebruikt aparte methods in de Update loop die omschrijven wat de logica doet. Ook plaats je comments om het nog beter uit te leggen.

Dit is een heel goed voorbeeld wat je ook bij de andere classes moet doen 💯